### PR TITLE
Event-user connection and displaying event details

### PR DESCRIPTION
**Pull request by QuaternionX**

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -23,6 +23,7 @@ def edit; end
   # POST /events.json
   def create
     @event = Event.new(event_params)
+    @event.owner_id = current_user.id;
```
Comment (ozimeu) [inline app/controllers/events_controller.rb]: `;` :)

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -59,6 +60,17 @@ def destroy
     end
   end
 
+  def join
+    respond_to do |format|
+      if current_user
+       @event.users << current_user
```
Comment (ozimeu) [inline app/controllers/events_controller.rb]: 1 space missing in the indentation

File: app/views/events/index.html.erb [hunk 3]
```diff
@@ -16,9 +19,16 @@
       <tr>
         <td><%= event.name %></td>
         <td><%= event.description %></td>
+        <td><%= event.owner_id %></td>
+        <td><%= event.users.count %></td>
+        <td><% event.users.each do |user| %>
+          <%= "#{user.email}" %><br/>
```
Comment (ozimeu) [inline app/views/events/index.html.erb]: wrong indentation

File: app/views/events/show.html.erb [hunk 4]
```diff
@@ -10,5 +10,14 @@
   <%= @event.description %>
 </p>
 
+
```
Comment (ozimeu) [inline app/views/events/show.html.erb]: extra empty line

File: app/views/events/show.html.erb [hunk 5]
```diff
@@ -10,5 +10,14 @@
...
+<p>
+  <strong>Uczestnicy:</strong>
+  <ul><% @event.users.each do |user| %>
+  <li><%= "#{user.email}" %></li>
```
Comment (ozimeu) [inline app/views/events/show.html.erb]: wrong indentation

**Review by ghost — APPROVED** [review]

**Pull request merged**